src/c_api.cc

```
@@ -58,6 +58,28 @@ class ReadCbStreamBuf : public std::streambuf {
   char buf_[64 * 1024];
 };
 
+// Parts kept in flight for a streaming upload.
+//
+// The default is 1, which uploads parts strictly one at a time: the next part
+// is not read until the previous one has been acknowledged, so the link sits
+// idle for a whole round trip per part. On an RDMA stream that measured
+// 167.7 MiB/s per caller against 318.6 at four-deep.
+//
+// It is not free. The parallel path allocates one part buffer per in-flight
+// part and registers each for RDMA, so the pinned memory is
```

Cppcheck (2.21.0)

[performance] Prefer prefix ++/-- operators for non-primitive types.

(postfixOperator)